charts/timed/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+Open following URL to start using Timed:
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.hosts }}
+  http://{{ . }}
```

Comment:
Check if this works better to automatically switch from `http://` to `https://`
```suggestion
  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ . }}
```

Reply:
changed